internal/venice-common/src/main/java/com/linkedin/venice/blobtransfer/BlobFinder.java

```diff
@@ -1,6 +1,9 @@
 package com.linkedin.venice.blobtransfer;
 
-public interface BlobFinder {
+import java.io.Closeable;
+
+
+public interface BlobFinder extends Closeable {
```

Comment:
nit: it's better to have AutoClosable

Reply:
thanks - why is AutoClosable better?

Reply:
https://stackoverflow.com/questions/13141302/implements-closeable-or-implements-autocloseable